src/planscape/eval/compute_conditions.py

```diff
@@ -143,7 +141,8 @@ def score_pillar(condition_reader: ConditionReader, pillar: Pillar, condition_ty
             element_score = score_element(condition_reader, element, condition_type, False)
         element_conditions.append(element_score)    
     operation = pillar.get('operation', 'MEAN')
-    return _summarize(element_conditions, operation if operation else 'MEAN')
+    # TODO: Parameterize the NoData value
+    return _summarize(np.finfo(np.float32).min,element_conditions, operation if operation else 'MEAN')
```

Comment:
Does the type checker like this?  I recall having to wrap the first arg in float(...)

Reply:
I think it's ok because the min function returns a floating point number